scanpy/external/pl.py

```diff
@@ -234,3 +239,443 @@ def sam(
         if colorbar:
             plt.colorbar(cax, ax=axes)
     return axes
+
+
+def wishbone_marker_trajectory(
+    adata: AnnData, no_bins: int = 150, smoothing_factor: int = 1,
+):
+    """\
+    Plot marker trends along trajectory
+
+    Parameters
+    ----------
+    adata
+        Annotated data matrix.
+    no_bins
+        Number of bins for calculating marker density
+    smoothing_factor
+        Parameter controlling the degree of smoothing
+
+    Returns
+    -------
+    Updates `adata` with the following fields:
```

Comment:
Hmm, why is something that updates adata in `pl` and not in `tl`?

Reply:
This code is adapted from Wishbone package and it is part of the plotting scheme within the class. I can definitely move it over to `tl._wishbone` and have it as a separate supporting code for the plotting. This code returns data that could be useful for plotting heatmaps down the road.